hyperjumptech/monika
# feat: add repeats and custom id sequences

## PR

### What this PR does
Implements #147 to add run options
1. Added -r  --repeat flag
2. Added -i  --id flag   
3. Add documentations in guides

### Implementation 
1. Added an id feeder to handle user specified probe.id sequences
2. Added counter checking for repeat control

## How to test
1. Running `monika` or `bin/run` should work normally
2. `monika -r x` where x is any number, should repeat just x times
3. `monika -i 1,2,3` should run probe id 1, 2 and then 3. Sequence can be any valid id sequence (ex: 1,2,3 or 3,2,1 or 1,2,1) 
4. `monika -r 3 -i 1,2` should run probe id 1 and 2 repeating 3 times
5. any invalid probe.id should be caught
